data_types/src/database_rules.rs

```diff
@@ -25,6 +25,8 @@ pub enum Error {
 
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 
+const DEFAULT_CHUNK_MOVER_CHECK_DURATION: std::time::Duration = std::time::Duration::from_secs(10); // 10 seconds
```

Comment:
I've seen comments like that go out of sync soon enough; after all, the compiler doesn't compile comments :-)

```suggestion
const DEFAULT_CHUNK_MOVER_CHECK_DURATION: std::time::Duration = std::time::Duration::from_secs(10);
```